osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-smr52.json

```diff
@@ -199,6 +199,54 @@
           "access": "RW"
         }
       ]
+    },
+    {
+      "tag": "ALARM_FILTER_1",
+      "description": "Alarm filter 1",
+      "note": null,
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.97.98.10.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "RW"
+        }
+      ]
+    },
+    {
+      "tag": "ALARM_FILTER_2",
```

Comment:
Ik zie dit object niet in de 5.2 specificatie staan, maar zie het pas in de 5.5 spec. Kun jij aangeven in welk 5.2 document jij dit object hebt gevonden?

Reply:
Ik heb inmiddels gevonden dat deze in de 5.2 SagSwell spec staat, dus object hoort inderdaad hier